pkg/stanza/adapter/register.go

```diff
@@ -15,7 +15,6 @@
 package adapter // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/adapter"
 
 import (
-	// Register parsers and transformers for stanza-based log receivers
```

Comment:
GCI removes comments from the import block, so this comment is lost.

Reply:
This is a pity :confused: personally I am fine with it, but could we report this upstream?

Reply:
FWIW that's the only comment it removes. If you place a comment inline to an import, it is preserved.